codegen/build.go

```diff
@@ -99,6 +106,21 @@ func (cfg *Config) resolver() (*ResolverBuild, error) {
 	}, nil
 }
 
+func (cfg *Config) server(destDir string) *ServerBuild {
+	imports := Imports{
+		destDir: destDir,
+	}
+	imports.add(cfg.Exec.ImportPath())
```

Comment:
you probably should include any imports that aren't covered by https://github.com/vektah/gqlgen/blob/master/codegen/import_build.go#L11-L27 here too (`http` and `githhub.com/vektah/gqlgen/handler`)

Without them, gqlgen can end up with duplicate package aliases in the generated code.

Reply:
hrmm I didn't encounter that issue but sure. Updated.